instaclustr/resource_cluster.go

```diff
@@ -673,6 +718,11 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 			}
 			rackList = appendIfMissing(rackList, node.Rack)
 		}
+
+		// for multi-DC provisioning
+		if len(cluster.DataCentres) > 1 {
+			break
+		}
 	}
 	rackCount := len(rackList)
 	nodesPerRack := nodeCount / rackCount
```

Comment:
[question] I know you this isn't part of this PR, but is this valid? is it possible to have unbalanced racks which have different numbers of nodes per rack. Also with multi-dc we can have different rack configurations on each DC? Looks like we expect this to always be an integer with `rackAllocation["nodes_per_rack"] = strconv.Itoa(nodesPerRack)`

Reply:
I believe it is possible to have unbalanced racks here. Nodes can be added via the dashboard to a single rack (although we recommend against it).

Reply:
thanks Will, working on a solution now.